index.html

```diff
@@ -0,0 +1,117 @@
+      <div class="main">
+        <div class="home-page"><h2>Home</h2></div>
+        <div class="posting-section">
+            <div class="whats-happening"><p>What's Happening?</p></div>
+            <div class="uploading">
+            <div class="media-upload">
+              <img src="./image-outline-icon.svg">
+            </div>
+            <div class="gif-upload">
+              <img src="./gif-file-icon.svg">
+            </div>
+            <div class="poll-upload">
+              <img src="./poll-icon.svg">
+            </div>
+            <div class="emoji-upload">
+              <img src="./cool-emoji-icon.svg">
+            </div>
+            <div class="schedule-upload">
+              <img src="./daily-schedule-icon.svg">
+            </div>
+          </div>
+          <!--Add div for button-->
+            <button class="tweet-button2">Tweet</button>
+       </div>
+       <hr>
+       <!--Main - last section feed1-->
+       <div class="feed">
+        <div class="feed-grid1-profile-pic">
+          <img src="./default-profile-picture-grey-female-icon.svg">
+        </div>
+        <div class="feed-grid1">
+          <div class="tweet-section-username"><p><strong>Priya</strong> <span>@sunshineandsmiles 33m</span></p></div>
+          <div class="feed-text"><p>@jellybaby @rainbows Live in the present.</p></div>
+          <div class="feed-icons"></div>
+         </div>
+
+     <!--Main - last section feed2-->
+     <div class="feed-grid2-profile-pic">
+      <img src="./default-profile-picture-grey-female-icon.svg">
+    </div>
+     <div class="feed-grid2">
+      <div class="tweet-section-username"><p><strong>Amber</strong> <span>@jellybabies 53m</span></p></div>
+      <div class="feed-text"><p>@sunshineandsmiles Get lost in what you love.</p></div>
+      <div class="feed-icons"></div>
+     </div>
+     <!--Main - last section feed3-->
+     <div class="feed-grid3-profile-pic">
+      <img src="./default-profile-picture-grey-male-icon.svg">
+    </div>
+     <div class="feed-grid3">
+      <div class="tweet-section-username"><p><strong>Sangeeta</strong> <span>@rainbowsandcherries 3h</span></p></div>
+      <div class="feed-text"><p>@sunshineandsmiles Be good. Do good.</p></div>
+      <div class="feed-icons"></div>
+     </div>
+       </div>
+      
+        <!--Closing Main Bar - Div below-->
+    </div>
+      <!--Main Bar Ends-->
+
+      <!--Right Side-Bar Starts-->
+      <div class="right-bar">
+        <div class="magnifying">
+              <img src="./magnifier-glass-icon (1).svg">
+            </div> 
+          <div class="search-twitter-bar">Search Twitter
+             
+          </div>
+          <div class="search-whats-happening"><h2>What's Happening</h2><p><span>Trending in United Kingdom     </span></p><p>Usyk</p><p>32.8K Tweets</p><p><span>Politics &#183 Trending</span></p><p>Council Tax</p><p>7,400 Tweets</p><p><span>News &#183 Trending</span></p><p>Owen Jones</p><p>2,054 Tweets</p></div>
```

Comment:
Is there a way to close gaps between paragraphs?

Reply:
line-height will allow you to change the spacing between lines

Reply:
Cheers Alex - I'll check that out